_posts/2024-01-17-geospatial-techno.md

```diff
@@ -0,0 +1,75 @@
+${GEOSERVER_DATA_DIR}/user_projections/epsg.properties
+- Configuration and catalog: This option is very useful to update the configuration without having to restart the service. If the configuration on the disk becomes outdated due to external changes, you can refresh it by loading the latest data from the disk.
+
+#### Module Field Descriptions
+In GeoServer, a module can fall into one of three classes:
+- Core, those modules that are visible by default in the Modules tab that GeoServer requires to function and are distributed with the main GeoServer distribution.
+- Extension, those modules that add functionality to GeoServer. They are installed as add-ons to the base GeoServer installation. After you download and install these extensions, they are added to the Geoserver modules list. For example WPS extension
+- Community, those modules that are generally considered experimental and are often under development. For that reason, Unlike the official extensions, these modules are not released and stored on SourceForge when an official GeoServer release is produced.
+
+Every module added to GeoServer has its origin as a community module. If the module becomes stable enough it will eventually become part of the main GeoServer distribution either as a core module or as an extension.
+
+#### System Status Field Descriptions
+System Status adds some extra information about the system in the GeoServer status page in a tab named System Status and makes that info queryable through the GeoServer REST interface. This info should allow an administrator to get a quick understanding of the status of the GeoServer instance. If the System status tab is not present, it means that the plugin was not installed correctly. The System status tab content will be refreshed automatically every second.
+
+### GeoServer Logs
+The "GeoServer Logs" page, lets you read the messages, warnings, and errors contained in the log file. According to the current logging settings, you can find more information about the requests clients send to GeoServer and how it processes them. You can only read the last 1,000 lines by default from the console. You can also change this setting, but if you need to access the entire log content, we would strongly suggest accessing it with a text editor.
+
+You can use the "Download the full log file" link placed just under the text console, or access the log file directly from this path: "geoserver_data_dir/logs/geoserver.log"
+
+### Contact Information
+The "Contact Information" page, is used in the Capabilities document of the WMS server and is publicly accessible. GeoServer provides an item to describe this information and metadata in different languages. By default, it's disabled and can be enabled from the i18n checkbox. You can complete this form with the relevant information and press the Save button to save your information.
```

Comment:
This is actually much more important now as the information is deployed on the main welcome page (providing an introduction to the service for your visitors).

Not sure if your video covers that?